### PR TITLE
The Betaflight blackbox explorer curves MinMax values control manual

### PR DESCRIPTION
**Pull request by demvlad**
The Betaflight blackbox explorer curves MinMax values control manual

### CODE REVIEW

**Comment by haslinghuis** [comment]
Looks good @demvlad - only suggestion I have at the moment is to have shorter title and headers

@nerdCopter @ctzsnooze

**Comment by demvlad** [comment]
Ok. Shorter:
"The blackbox explorer curves MinMax control feature"
or maybe
"The blackbox explorer curves MinMax control"

**Comment by haslinghuis** [comment]
Perhaps `Blackbox explorer MinMax control curves` 

Can we have some advice from native english speakers here :joy:

**Comment by nerdCopter** [comment]
shortest: `Blackbox Explorer MinMax control.`
moderate: `Blackbox Explorer MinMax curve control.`
but more precisely, `Blackbox Explorer MinMax curve scale control.` or `Blackbox Explorer MinMax control for curve scales` 

IMHO / open to options.  
Shortest is fine by me, then users learn the details in the docs.

**Comment by demvlad** [comment]
> `SUGGESTED`:

Ok. Will look. Thank's for help.

**Comment by demvlad** [comment]
> Suggestions for remaining menu items:

Accepted.

**Review by haslinghuis — DISMISSED** [review]
LGTM!

**Comment by demvlad** [comment]
> LGTM!

one addition :)! The bookmarks.

**Review by haslinghuis — DISMISSED** [review]

File: docs/wiki/guides/current/Black-Box-logging-and-usage.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ Some features are automatically disabled depending upon the firmware you are run
 
 Do click on all the buttons to learn what they do and '?' for the Keyboard Short cuts.
 
+#### Betaflight blackbox explorer Version 3.7
+
+[The charts curves MinMax values control feature](/docs/wiki/guides/current/BBE-MinMax-control-manual)
```
Comment (nerdCopter) [inline docs/wiki/guides/current/Black-Box-logging-and-usage.md:44]:
```suggestion
[Blackbox Explorer MinMax control](/docs/wiki/guides/current/BBE-MinMax-control-manual)
```

Comment (nerdCopter) [reply docs/wiki/guides/current/Black-Box-logging-and-usage.md:44]: maybe :man_shrugging:

Comment (demvlad) [reply docs/wiki/guides/current/Black-Box-logging-and-usage.md:44]: Short sentence. Ok.

**Review by nerdCopter — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**